hips/hip-0001.md

```diff
@@ -0,0 +1,427 @@
+idea. The proposal champion (a.k.a. author) should first attempt to ascertain
+whether the idea is proposal-able. Posting to the [cncf-helm][] mailing list is
+the best way to go about this.
+
+Vetting an idea publicly before going as far as writing a proposal is meant to
+save the potential author time. Many ideas have been brought forward for
+changing Helm that have been rejected for various reasons. Asking the Helm
+community first if an idea is original helps prevent wasting time on something
+that is guaranteed to be rejected based on prior discussions (searching the
+internet does not always do the trick). It also helps to make sure the idea is
+**applicable to the entire community and not just the author**. Just because an
+idea sounds good to the author does not mean it will work for most Helm users in
+most areas where Helm is used.
+
+Once the champion has asked the Helm community as to whether an idea has any
+chance of acceptance, a summary of the proposal should be presented to
+[cncf-helm][]. This gives the author a chance to flesh out the proposal as a
+high quality, well-formatted proposal, and to address initial concerns about the
+proposal.
+
+### Submitting a HIP
+
+Following a discussion on [cncf-helm][], the proposal should be submitted as a
+proposal via a [GitHub pull request][pr]. The proposal must be written in
+proposal style as described below, else it will fail review immediately
+(although minor errors may be corrected by the maintainers).
+
+The standard HIP workflow is:
+
+- You, as the proposal author, fork the helm/community repository and create a
+  new proposal
+- Proposals are written in [Markdown][markdown]
+- Push your proposal to your GitHub fork and [submit a pull request][pr].
+
+Once the review process is complete, the reviewers will approve the proposal by
+commenting on the proposal with a "Looks good to me" ("LGTM" for short) and
+merge the proposal. (note that this is not the same as accepting your proposal!)
+
+The reviewers will not unreasonably deny a proposal. Reasons for denying
+proposal status include duplication of effort, being technically unsound, not
+providing proper motivation, not addressing backwards compatibility, or not in
+keeping with Helm's design. The core maintainers can be consulted during the
+approval phase.
+
+Once approved, proposal authors are responsible for collecting community
+feedback on a proposal before submitting a feature that satisfies the proposal
+for review. However, wherever possible, long open-ended discussions on public
+mailing lists should be avoided. Strategies to keep the discussions efficient
+include: setting up a separate meetings for the topic, having the proposal
+author accept private comments in the early design phases, setting up a wiki
+page, etc. proposal authors should use their discretion here.
+
+## What belongs in a successful HIP
+
+Each HIP should include the following parts:
```

Comment:
Do we allow the proposal to contain other parts beyond the ones enumerated here?

Reply:
yes.